test/fw/ptl/utils/pbs_dshutils.py

```diff
@@ -190,11 +191,46 @@ def get_platform(self, hostname=None, pyexec=None):
                 _msg = 'Unable to retrieve platform info,'
                 _msg += 'defaulting to local platform'
                 self.logger.warning(_msg)
-                platform = sys.platform
+                splatform = sys.platform
             else:
-                platform = ret['out'][0]
-        self._h2p[hostname] = platform
-        return platform
+                splatform = ret['out'][0]
+        self._h2p[hostname] = splatform
+        return splatform
+
+    def get_platform_uname(self, hostname=None, pyexec=None):
```

Comment:
Can we have platform module related function calls in one method . get_os_info is called may at one other place i.e, snaputils . If its okay to change that . May be we can have one method and call function calls to platform as args.

Reply:
Based on further review comments from other reviewers, since dshUtils is a wrapper, it is suggested to have individual method for each function call. Hence, retained this method with new name as get_uname()